README.md

```diff
@@ -21,6 +21,30 @@ Top-level
  - ffi - uses Mozilla [uniffi](https://github.com/mozilla/uniffi-rs) to create native libraries with FFI bindings
    - Currently supports Android via Kotlin
 
+## Diagram
+
+```
+    CORE LIBRARY     │       BINDINGS          │    PLATFORM SDKs    │     APP
+─────────────────────┼─────────────────────────┼─────────────────────┼──────────────────
+ xmtp/libxmtp/crates │ xmtp/libxmtp/bindings   │                     │
+─────────────────────┼─────────────────────────┼─────────────────────┼──────────────────
+    protocol         │      protocol/dx        │    dx               │    devs
```

Comment:
I'm not sure DX would have any meaning to most people. Can these titles be changed to reflect the POV of all developers?

Reply:
Let me know how you feel about the latest, or just suggest some titles and I'll pop them right in!